content/tutorials/style_inbo_bibliography/index.Rmd

```diff
@@ -0,0 +1,93 @@
+---
+title: "Use of INBO citation style with bibliography managers"
+description: "We show where to find and how to use the INBO citation style in bibliography managers"
+author: "Hans Van Calster, Herwig Borremans, Floris Vanderhaeghe"
+date: "`r Sys.Date()`"
+categories: ["styleguide"]
+tags: ["bibliography", "csl", "INBOmd", "open science", "zotero", "mendeley", "endnote"]
+bibliography: "../../articles/reproducible_research.bib"
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+# The INBO citation style
+
+Most journals, publishers and companies use a specified format for citations and bibliographies.
+The Research Institute for Nature and Forest (INBO) is no exception.
+
+All bibliography managers provide ways to automate the formatting of citations and bibliographies which greatly facilitates scientific publishing.
+Depending on the bibliographic manager software, the style formatting rules are stored in specific files.
+In this tutorial you will learn how to use the INBO citation style in EndNote, Mendeley and Zotero bibliography managers.
+
+EndNote uses a `*.ens` file to store the style formatting rules, whereas Mendeley and Zotero use the open source `*.csl` file (Citation Style Language)[https://citationstyles.org/].
+
+The latest official CSL file of the INBO citation style is available at the CSL style repository (also transferred automatically to the Zotero style repository).
+See (finding-and-installing-styles)[https://citationstyles.org/authors/#/finding-and-installing-styles] for more information.
```

Comment:
Once the brackets are corrected, this link will render as:

> See [finding-and-installing-styles](https://citationstyles.org/authors/#/finding-and-installing-styles) for more information.

I'm not sure if that was your intention. (It might be)